setup.py

```diff
@@ -460,7 +460,7 @@ def setup_package():
 
     from setuptools import setup
```

Comment:
Perhaps move this to the `else` instead of the `noqa` line?

Reply:
this is a curious one. I tried this edit, however `python setup.py build bdist_wheel` fails, and I have no idea why. So I've given up and just marked the line with `noqa` instead. However, I'm open for input as to why.

Reply:
Ah, that's because `setuptools` patches `distutils` - so perhaps instead change to
```
if run_build:
    # patches distutils, even though we don't use it
    import setuptools  # noqa: F401
    from numpy.distutils.core import setup
    ...
else:
   from setuptools import setup
```

Reply:
Perfect, thanks!